### PR TITLE
Origin/feature/service check view update

### PR DESCRIPTION
**Pull request by pankroscog**

### CODE REVIEW

File: cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const ServiceCheckWidget = props => {
     ? 'OK'
     : body.includes(expectedResponseBody)
     ? 'MATCH'
-    : 'NO MATCH';
+    : 'NOT MATCH';
```
Comment (tMaxx) [inline cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js]: Come on, previous was correct :P

Comment (pankroscog) [reply cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js]: True, fixed

File: cogboard-webapp/src/components/styled/index.js [hunk 2]
```diff
@@ -34,6 +34,11 @@ export const Caption = props => (
 
 export const CaptionWithPointer = styled(Caption)`
   cursor: pointer;
+  text-align: center;
+  :hover {
```
Comment (goeson00) [inline cogboard-webapp/src/components/styled/index.js]: This works without &:hover?

File: cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js [hunk 3]
```diff
@@ -62,7 +66,7 @@ const ServiceCheckWidget = props => {
         </WidgetButton>
       </Caption>
       <CaptionWithPointer title={body} onClick={handleClick}>
-        Response: {bodyMessage}
+        ⯆ Response: {bodyMessage}
```
Comment (goeson00) [inline cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js]: ⯆ - what is this?

Comment (pankroscog) [reply cogboard-webapp/src/components/widgets/types/ServiceCheckWidget.js]: Removed (it looks like on some devices don't recognize this sign)

File: cogboard-webapp/src/components/styled/index.js (hunk 2 above)
Comment (pankroscog) [reply cogboard-webapp/src/components/styled/index.js]: Yes

Comment (pankroscog) [reply cogboard-webapp/src/components/styled/index.js]: Added & for better readibility

**Pull request merged**